ddl/db_test.go

```diff
@@ -6606,6 +6606,7 @@ func (s *testSerialDBSuite) TestIssue22819(c *C) {
 	tk1.MustExec("update t1 set v = 2 where v = 1")
 
 	tk2.MustExec("alter table t1 truncate partition p0")
+	tk2.MustExec("alter table t1 truncate partition all")
```

Comment:
I think you should add a standalone test, which does not interfere with this test.

Reply:
Agreed, please add a test case named like 'TestTruncateAllPartitions' and check the the result of this syntax.

Thanks!

Reply:
Agreed, please add a test case named like 'TestTruncateAllPartitions' and check the the result of this syntax.

Thanks!

Reply:
done, please take a look